shipyard_blueprints/redshift/pyproject.toml

```diff
@@ -15,8 +15,11 @@ shipyard-templates = "0.4.3"
 sqlalchemy-redshift = "0.8.14"
 shipyard-bp-utils = "0.1.1"
```

Comment:
bump version

Reply:
this will be removed - this is for the redshift branch